index.html

```diff
@@ -34,5 +33,187 @@ <h1 class="header" >I’m Mariane<br>
+                <div class="image_section2">
+                    <img src="image/Snapshoot Portfolio.svg" alt="Snapshoot Portfolio">
+                </div>
+                <h2 class="tonic">Tonic</h2>
+                <div class="flex-container">
+                    <ul>
+                    <li class="blue">CANOPY</li>
+                    <li><img src="image/Counter.svg" alt="Counter"></li>
+                    <li class="blue">Back End Dev</li>
+                    <li><img src="image/Counter.svg" alt="Counter"></li>
+                    <li class="blue">2015</li>
+                    </ul>
+                </div>
+                <article>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                <div class="tags">
+                    <ul>
+                    <li class="tag1"><img src="image/Tag_html.svg" alt="HTML Tag"></li>
+                    <li class="tag2"><img src="image/Tag_css.svg" alt="HTML Tag"></li>
+                    <li class="tag3"><img src="image/Tag.svg" alt="HTML Tag"></li>
+                    </ul>
+                </div>   
+                </article>
+                <nav class="nav-button">
+                    <a href="#"><input type="button" class="mybutton" value="See Project"></a> 
+                </nav>    
+        </section>
+
+        <section class="border_section3">
+                <div class="image_section3">
+                    <img src="image/Availability.svg" alt="Snapshoot Portfolio Availability">
+                </div>
+            <header class="border_heading">
+                <h2 class="heading">Multi-Post<br>
+                    Stories</h2>
+                <div class="flex-container">
+                    <ul>
+                    <li class="blue">CANOPY</li>
+                    <li><img src="image/Counter.svg" alt="Counter"></li>
+                    <li class="blue">Back End Dev</li>
+                    <li><img src="image/Counter.svg" alt="Counter"></li>
+                    <li class="blue">2015</li>
+                    </ul>
+                </div>
+            </header>
+            <article>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                <div class="tags">
+                    <ul>
+                    <li class="tag1"><img src="image/Tag_html.svg" alt="HTML Tag"></li>
+                    <li class="tag2"><img src="image/Tag_css.svg" alt="HTML Tag"></li>
+                    <li class="tag1"><img src="image/Tag.svg" alt="HTML Tag"></li>
+                    </ul>
+                </div>
+            </article>
+                <nav class="nav-button">
+                    <a href=""><input type="button" class="mybutton" value="See Project"></a> 
+                </nav>
```

Comment:
- [ ] I would suggest to not use `nav` tag here for semantics reason. Here is not the proper place for its usage. You can simply use `button` tag to wrap your `See Project` text or use  a link (`a`) tag. Feel free to use any of the two suggestions